novoda-blog-diversity/lib/data-source.js

```diff
@@ -5,9 +5,9 @@ const androidTags = ["android", "google", "kotlin", "android studio", "gradle"]
 const iosTags = ["ios", "xcode", "swift", "apple", "cocoa pods"]
 
 const generateViewStateFrom = (url) => {
-    return parseRss(url).then(rss => {
-        return rss.feed.entries;
-    }).then(toViewState)
+    return parseRss(url)
+        .then(rss => rss.feed.entries)
```

Comment:
When a function body consist only in 1 line with a `return` you can inline it like this